package.json

```diff
@@ -27,6 +27,8 @@
     "test:eslint-rules": "node eslint-rules/*.test.mjs",
     "test:trace:core": "tap packages/dd-trace/test/*.spec.js \"packages/dd-trace/test/{ci-visibility,datastreams,encode,exporters,opentelemetry,opentracing,plugins,service-naming,standalone,telemetry}/**/*.spec.js\"",
     "test:trace:core:ci": "npm run test:trace:core -- --coverage --nyc-arg=--include=\"packages/dd-trace/src/**/*.js\"",
+    "test:trace:guardrails": "mocha -r \"packages/dd-trace/test/setup/mocha.js\" \"packages/dd-trace/test/guardrails/**/*.spec.js\"",
```

Comment:
The folder will also need to be added to the exclusions for core tests, otherwise they will run there too.

Reply:
Actually it looks like we switched to a whitelist, so never mind.